src/InfluxDB2/UdpWriter.php

```diff
@@ -84,15 +84,35 @@ protected function writeSocket($payload)
     /**
      * Create (if not exists) socket to write UDP datagrams
      * @return false|resource
+     * @throws \Exception
      */
     protected function getSocket()
     {
         if (empty($this->socket)) {
-            $this->socket = socket_create(AF_INET, SOCK_DGRAM, SOL_UDP);
+
+            $this->socket = socket_create($this->getConfiguredInetVersion(), SOCK_DGRAM, SOL_UDP);
```

Comment:
Please remove blank line to satisfy code style check:

```suggestion
            $this->socket = socket_create($this->getConfiguredInetVersion(), SOCK_DGRAM, SOL_UDP);
```

Reply:
Will do 👍